hack/make-rules/test-cmd.sh

```diff
@@ -2324,6 +2324,26 @@ __EOF__
   kube::test::get_object_assert "nodes 127.0.0.1" "{{.spec.unschedulable}}" '<no value>'
 
 
+
+  ########################
+  # authorization.k8s.io #
+  ########################
+
+  # check remote authorization endpoint, kubectl doesn't actually display the returned object so this isn't super useful
+  # but it proves that works
```

Comment:
> I don't know that we want a ton of output on kubectl create by default.  It
> will be extremely difficult to take a random list of objects and make a
> usable, effective output.  We'd probably want to err on a very succinct
> result, and I doubt our ability to do that generically.

I'd keep the flag off by default, but if you request something like `-o yaml`, I think you expect to see your output, not your input.